### ISSUE TITLE
Transition skip to link onto page when receiving focus

### ISSUE THREAD

**Issue by SMaxOwok** [ManifoldScholar/manifold#2009]
>Skip nav lacks focus indicator

I think the best way to address this is to have the skip link off the page and transition it on when it receives focus.

https://webaim.org/techniques/skipnav/#invisible
